internal/podman/ssh_test.go

```diff
@@ -0,0 +1,232 @@
+package podman
+
+import (
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/testdata"
+)
+
+func TestNewSSHConfig(t *testing.T) {
+	paths := []string{
+		"./test-fixtures/containers.conf",
+		"./test-fixtures/empty.conf",
+	}
+
+	const (
+		sshAddress = "ssh://core@localhost:45983/run/user/1000/podman/podman.sock"
+		sshKeyPath = "/home/jonas/.ssh/podman-machine-default"
+	)
+
+	address, identity := getSSHAddress(paths)
+	assert.Equal(t, sshAddress, address)
+	assert.Equal(t, sshKeyPath, identity)
+
+	conf, err := newSSHConf(address, identity, "")
```

Comment:
no request for change here, but you could do `cmp.Equal` to compare whole structs against expected structs, so you can do something like:
```golang
expected := mystruct{
  Value1: "thing"
  Value2: OtherStruct{
      Value3: "3",
   },
}

assert.True(t, cmp.Equal(expected, actual))

```